translations/ga/census_household_gb_nir_ga.po

```diff
@@ -2,4301 +2,6502 @@ msgid ""
+msgid "Has <em>{household_address}</em> been designed or adapted for any of the following?"
+msgstr "Ar dearadh nó ar oiriúnaíodh <em>{household_address}</em> do cheann ar bith de na rudaí seo a leanas?"
 
-msgid "s"
-msgstr "s"
+#. Question text
+msgid "What type of central heating does <em>{household_address}</em> have?"
+msgstr "Cén sórt teasa lárnaigh atá ag <em>{household_address}</em>?"
 
-msgid "e"
-msgstr "e"
+#. Question text
+msgid "What type of renewable energy systems does <em>{household_address}</em> have?"
+msgstr "Cén sórt córas fuinnimh in-athnuaite atá ag <em>{household_address}</em>?"
 
-msgid "l"
-msgstr "l"
+#. Question text
+msgid "Does your household own or rent <em>{household_address}</em>?"
+msgstr "An le do líon tí <em>{household_address}</em>nó an dtógann sé ar cios é?"
 
-msgid "d"
-msgstr "d"
+#. Question text
+msgid "Who do you rent from?"
+msgstr "Cé óna bhfaigheann tú an réadmhaoin ar cíos?"
 
-msgid "People who work away from home within the UK if this is their permanent or family home"
-msgstr "Daoine a oibríonn ar shiúl ón bhaile taobh istigh den UK más é seo a dteach buan nó teaghlaigh"
+#. Question text
+msgid "In total, how many cars or vans are owned, or available for use, by members of this household?"
+msgstr "San iomlán, cá mhéad gluaisteán nó veain ar le baill den líon tí seo iad nó atá ar fáil le húsáid acu?"
 
-msgid "Members of the armed forces if this is their permanent or family home"
-msgstr "Baill de na fórsaí armtha más é seo a dteach buan nó teaghlaigh"
+#. Question text
+msgid "Are you <em>{person_name}?</em>"
+msgstr "An tusa <em>{person_name}?</em>"
 
-msgid "People who are temporarily outside the UK for less than <strong>12 months</strong>"
-msgstr "Daoine atá taobh amuigh den UK go sealadach do níos lú ná <strong>12 mhí</strong>"
+#. Question text
+msgid "What is your date of birth?"
+msgstr "Cad é do dháta breithe?"
 
-msgid "Other people who usually live here but are temporarily away"
-msgstr "Daoine eile a chónaíonn anseo de ghnáth ach atá ar shiúl go sealadach"
+#. Question text
+msgid "What is <em>{person_name_possessive}</em> date of birth?"
+msgstr "Cad é dáta breithe <em>{person_name_possessive}</em>?"
 
-msgid "People staying temporarily who usually live in the UK but do not have another UK address for example, relatives, friends"
-msgstr "Daoine a bheidh ag fanacht go sealadach a chónaíonn sa UK ach nach bhfuil seoladh ar bith eile acu sa UK, mar shampla, gaolta, cairde"
+#. Question text
+msgid "You are {age} old. Is this correct?"
+msgstr "Tá tú {age} d'aois. An bhfuil sé seo ceart?"
 
-msgid "People who usually live outside the UK who are staying in the UK for <strong>3 months or more</strong>"
-msgstr "Daoine a chónaíonntaobh amuigh den UK de ghnáth atá ag fanacht sa UK ar feadh <strong>3 mhí nó níos mó</strong>"
+#. Question text
+msgid "<em>{person_name}</em> is {age} old. Is this correct?"
+msgstr "Tá <em>{person_name}</em> {age} bliain d'aois. An bhfuil sé seo ceart?"
```

Comment:
I think "bliain" needs to be removed here as the word for "years" is added by runner.

Reply:
changed